scopes/react/react/webpack/webpack.config.base.ts

```diff
@@ -47,6 +44,13 @@ export default function (isEnvProduction = false): Configuration {
   // passed into alias object. Uses a flag if passed into the build command
   const isEnvProductionProfile = process.argv.includes('--profile');
 
+  const baseStyleLoadersOptions = {
+    miniCssExtractPlugin: isEnvProduction ? MiniCssExtractPlugin.loader : styleLoaderPath,
```

Comment:
change this name of the arg in that case

Reply:
updated